torch/linalg/__init__.py

```diff
@@ -258,44 +269,49 @@
 
-For a square matrix :math:`A`, this is defined as
+Letting :math:`\mathbb{K}` be :math:`\mathbb{R}` or :math:`\mathbb{C}`,
+the eigenvalue decomposition of a square matrix
+:math:`A \in \mathbb{K}^{n \times n}` is defined as
 
 .. math::
 
-    A = V \operatorname{diag}(L) V^{-1}
+    A = V \operatorname{diag}(\Lambda) V^{-1}\mathrlap{\qquad V \in \mathbb{C}^{n \times n}, \Lambda in \mathbb{C}^n}
 
-This decomposition exists if and only if :math:`A` is `diagonalizable`_. This is the case when all its eigenvalues are different.
+This decomposition exists if and only if :math:`A` is `diagonalizable`_.
+This is the case when all its eigenvalues are different.
 
-The returned decomposition is a named tuple `(eigenvalues, eigenvectors)`,
-which corresponds to :math:`L`, :math:`V` above. If :attr:`input` is a batch of matrices,
-then `L`, `V` are also batched with the same batch dimensions.
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
 
-Supports :attr:`input` of float, double, cfloat and cdouble dtypes.
-The output tensors `eigenvalues` and `eigenvectors` will always be complex-valued, even when :attr:`input` is real.
+The returned decomposition is a named tuple `(eigenvalues, eigenvectors)`,
+which correspond to :math:`\Lambda` and :math:`V` above.
+The output tensors `eigenvalues` and `eigenvectors` will always be complex-valued, even when :attr:`A` is real.
 
 .. note:: The eigenvalues and eigenvectors of a real matrix may be complex.
 
 .. note:: This function is computed using LAPACK's and MAGMA's `geev` for CPU and CUDA inputs,
           respectively.
           For CUDA inputs, this function synchronizes that device with the CPU.
 
-.. note:: The eigenvectors of a matrix are not unique, as multiplying an eigenvector by a
-          non-zero number produces another set of eigenvectors for the matrix.
-          As such, the returned eigenvectors are normalized to have norm
-          `1` and largest real component.
-          In particular, the eigenvectors are not continuous functions of the inputs.
+.. warning:: The eigenvectors of a matrix are not unique, as multiplying an eigenvector by a
+             non-zero number produces another set of valid eigenvectors of the matrix.
+             As such, the returned eigenvectors are normalized to have norm
+             `1` and largest real component.
+             As this choice is not canonical, it happens that the eigenvectors are not continuous
+             functions of the inputs. For this reason, different platforms, like NumPy, or inputs
+             on different devices, may produce different singular vectors.
 
-.. warning:: This function assumes that :attr:`input` is `diagonalizable`_ (e.g. when all the
+.. warning:: This function assumes that :attr:`A` is `diagonalizable`_ (e.g. when all the
```

Comment:
"e.g. when all the" -> "i.e. that its eigenvalues are unique"

Reply:
Will keep the wording for now, as per https://github.com/pytorch/pytorch/pull/56265#discussion_r619168317